SerialPrograms/Source/PokemonSV/Programs/Boxes/PokemonSV_BoxRoutines.cpp

```diff
@@ -123,6 +123,8 @@ void change_view_to_judge(
         //  If less than 4 of the IVs are read, assume we're not on the judge screen.
         if (detected < 4){
             pbf_press_button(context, BUTTON_PLUS, 20, 230);
+        }else if (detected == 6){
+            break;

```

Comment:
Change it to this:
```
}else{
}
```

The original code intended to succeed if at least 4 were read correctly. Since in some languages, reading is reliable. If we read at least 4 IVs correctly, we're certain to be on the IV judge.